File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.js
function solution(quiz) {
    return quiz.map(item => {
        let [left, right] = item.split("=").map(part => part.trim());

        let leftValue = new Function(`return ${left}`)();
        let rightValue = Number(right); 

        return leftValue === rightValue ? "O" : "X";
    });
}

File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.py
def solution(quiz):
    answer = []
    for q in quiz:
        a,ob,b,_,result = q.split(' ')
        a = int(a)
        b = int(b)
        result = int(result)
        if(ob == '+'):
            num = a + b
        else:
            num = a - b
        if(num == result):
            answer.append("O")
        else:
            answer.append("X")
    return answer